frontend/www/js/omegaup/components/group/Identitiesv2.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <div class="upload-csv">
+        <div>
+          <omegaup-markdown :markdown="T.groupsCsvHelp"></omegaup-markdown>
+          {{ T.groupsUploadCsvFile }}
+          <input name="identities" type="file" @change="readCsv" />
+        </div>
+      </div>
+      <br />
+      <div v-show="identities.length > 0" class="card no-bottom-margin">
+        <div class="card-header">
+          <h3 class="card-title">{{ T.wordsIdentities }}</h3>
+        </div>
+        <table class="identities-table table">
```

Comment:
nit:

```suggestion
        <table class="table" data-identities-table>
```

si se necesita para Selenium. si no, se puede eliminar?

Reply:
Si se va a utilizar, aunque sea hasta que terminemos de migrar todas las partes.